src/ts/ConvivaAnalytics.ts

```diff
@@ -370,52 +400,59 @@ export class ConvivaAnalytics {
         break;
     }
 
+    // TODO: check if also should not report for post-roll (currently doesn't)
     if (!this.isValidSession()) {
       // Don't report without a valid session (e.g. in case of a post-roll ad)
       return;
     }
 
-    this.debugLog('adstart', event);
+    this.debugLog('adbreakstart', event);
     this.client.adStart(this.sessionKey, Conviva.Client.AdStream.SEPARATE, Conviva.Client.AdPlayer.CONTENT, adPosition);
     this.onPlaybackStateChanged();
   };
 
-  private onAdSkipped = (event: any) => {
-    this.onCustomEvent(event);
-    this.onAdFinished(event);
-  };
+  private getAdBreakPosition(adBreak: AdBreak) {
```

Comment:
I would suggest do make this to something like `mapAdPosition` and return the conviva enum value. So we don't have to compare the strings afterwards again.